monai/handlers/mlflow_handler.py

```diff
@@ -115,14 +182,53 @@ def attach(self, engine: Engine) -> None:
             engine.add_event_handler(Events.ITERATION_COMPLETED, self.iteration_completed)
         if self.epoch_log and not engine.has_event_handler(self.epoch_completed, Events.EPOCH_COMPLETED):
             engine.add_event_handler(Events.EPOCH_COMPLETED, self.epoch_completed)
+        if not engine.has_event_handler(self.complete, Events.COMPLETED):
+            engine.add_event_handler(Events.COMPLETED, self.complete)
 
-    def start(self) -> None:
+    def start(self, engine: Engine) -> None:
         """
         Check MLFlow status and start if not active.
 
         """
+        mlflow.set_experiment(self.experiment_name)
         if mlflow.active_run() is None:
-            mlflow.start_run()
+            run_name = f"run_{time.strftime('%Y%m%d_%H%M%S')}" if self.run_name is None else self.run_name
+            mlflow.start_run(run_name=run_name)
+
+        if self.experiment_param:
+            mlflow.log_params(self.experiment_param)
+
+        attrs = {attr: getattr(engine.state, attr, None) for attr in self.default_attr_name}
```

Comment:
Can we allow user to specify other attrs of `engine.state.XXX` to log except the default ones?

Thanks.

Reply:
Yes, this line is for some default paramerters. Users can specified their own attrs throught the line below.
https://github.com/binliunls/MONAI/blob/cabc6cc56e78cf54c640ec3e22efa35eb6dc7da1/monai/handlers/mlflow_handler.py#L286
Thanks
Bin

Reply:
I would suggest to put the `["max_epochs", "epoch_length"]` as default value of the `state_attributes` arg.
What do you think?

Thanks.